apps/angular-console/src/app/app.component.ts

```diff
@@ -50,10 +50,19 @@ export class AppComponent implements OnInit, OnDestroy {
     settings: Settings,
     private readonly contextualActionBarService: ContextualActionBarService,
     private readonly titleService: Title,
-    // Ensures we allow location ext to listen on all router events no matter which route user enters into first.
-    private readonly routerNavService: RouterNavigationService
+    router: Router
   ) {
     settings.fetch().subscribe();
+
+    router.events
+      .pipe(filter((e): e is NavigationEnd => e instanceof NavigationEnd))
+      .subscribe(e => {
+        if (e.urlAfterRedirects === '/connect') {
+          this.contextualActionBarService.breadcrumbs$.next([
+            { title: 'Connect' }
```

Comment:
why is this needed?

Reply:
Its needed only because the narwhal extension do not set their own breadcrumbs. I'll mark it with a todo to move the logic over to the other repo